src/main/osd/osd_elements.c

```diff
@@ -921,16 +921,27 @@ static void osdElementOsdProfileName(osdElementParms_t *element)
 }
 #endif
 
-#ifdef USE_ESC_SENSOR
+#if defined(USE_ESC_SENSOR) || defined(USE_DSHOT_TELEMETRY)
+
 static void osdElementEscTemperature(osdElementParms_t *element)
 {
     if (featureIsEnabled(FEATURE_ESC_SENSOR)) {
         tfp_sprintf(element->buff, "E%c%3d%c", SYM_TEMPERATURE, osdConvertTemperatureToSelectedUnit(osdEscDataCombined->temperature), osdGetTemperatureSymbolForSelectedUnit());
+    } else {
```

Comment:
```suggestion
#if defined(USE_ESC_SENSOR) || defined(USE_DSHOT_TELEMETRY)

static void osdElementEscTemperature(osdElementParms_t *element)
{
#if defined(USE_ESC_SENSOR)
    if (featureIsEnabled(FEATURE_ESC_SENSOR)) {
        tfp_sprintf(element->buff, "E%c%3d%c", SYM_TEMPERATURE, osdConvertTemperatureToSelectedUnit(osdEscDataCombined->temperature), osdGetTemperatureSymbolForSelectedUnit());
    } else
#endif
    {
```

Reply:
Fixed

Reply:
@damosvil still needs some work.

In `target/common_pre.h` please comment out line 37 (USE_DHOT_TELEMETRY) and line 296 (USE_ESC_SENSOR) and try to `make STM32F411` so you can see what I mean.

Tested on `master` there is no problem with compilation and linking.

Reply:
Ok, I will try assap. Sorry I didn't understand before

Reply:
Reworked

Reply:
Thanks. It works if both are commented out, but we we also need to be able to disable the define independent. Almost there.

Reply:
Fixed